src/Symfony/Component/Uuid/Tests/UuidTest.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Tests\Component\Uuid;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Uuid\Uuid;
+
+class UuidTest extends TestCase
+{
+    private const AN_UUID_V1 = 'd9e7a184-5d5b-11ea-a62a-3499710062d0';
+    private const AN_UUID_V4 = 'd6b3345b-2905-4048-a83c-b5988e765d98';
+
+    public function testConstructorWithInvalidUuid()
+    {
+        $this->expectException(\InvalidArgumentException::class);
+        $this->expectExceptionMessage('The UUID is not valid.');
+
+        new Uuid('this is not an uuid');
+    }
+
+    public function testConstructorWithValidUuid()
+    {
+        $uuid = new Uuid(self::AN_UUID_V4);
+
+        $this->assertSame(self::AN_UUID_V4, (string) $uuid);
+        $this->assertSame('"'.self::AN_UUID_V4.'"', json_encode($uuid));
+    }
+
+    /**
+     * @requires extension uuid
```

Comment:
Why is the extension required here?

Reply:
It's required because the extension is not installed on appveyor. So the polyfill is used.
And the polyfill does not work on 32 bits system. 
So it's a lazy way to make test pass